terragrunt/aws/alert_compromise/functions/broadcast_alert.py

```diff
@@ -69,4 +75,15 @@ def lambda_handler(event, context):
                 Subject=os.environ["subject"],
             )
             print("Alert published for one event...")
-    print("Done processing all events...")
\ No newline at end of file
+    print("Done processing all events...")
+
+
+def does_key_contain_all_zeros(key):
+    # Split the key by the delimiter (e.g., '-')
+    parts = key.split('-')[-5:]
+
+    # Check if all parts that are numeric are zeros
+    if all(part.isdigit() and part == '0' * len(part) for part in parts if part.isdigit()):
+        return True
+    else:
+        return False
```

Comment:
Would this simplified version work? I haven't run it against the unit tests but I think it's equivalent:
```suggestion
    return all(part == '0' * len(part) for part in parts if part.isdigit())
```